packages/core/src/extensions/clipboardTextSerializer.ts

```diff
@@ -32,6 +38,20 @@ export const ClipboardTextSerializer = Extension.create<ClipboardTextSerializerO
             const textSerializers = getTextSerializersFromSchema(schema)
             const range = { from, to }
 
+            if (isCellSelection(view.state.selection)) {
+              const contentArray = (content.content as _Fragment).content
+              let result = ''
+
+              contentArray.forEach((tableRow:Node) => {
+                const cellArr = (tableRow.content as _Fragment).content
+
+                cellArr.forEach((cell:Node, cellIndex:Number) => {
+                  result = `${result}   ${cell.textContent}   ${Number(cellIndex) === cellArr.length - 1 ? '\n' : ''}`
+                })
+              })
+              return result
+            }
+
```

Comment:
This doesn't seem like the correct place for this, for example, where is the logic for adding these new lines for things like list items that you'd expect a newline between too?

Maybe they need custom [renderText methods](https://github.com/ueberdosis/tiptap/blob/07f4c03315f91b1c597ff764b395a488514a9036/packages/core/src/Node.ts#L684-L705)?

Reply:
> list items

I didn't take into account the situation where the table contains list items. I will reconsider

Reply:
Not only when it contains list items, but any content that can be broken into lines.

I think that you need a custom renderText method for table rows that allow you to insert line breaks or something.

But I don't completely understand what the expected outcome should be here, a table is fundamentally very different from plain text so it would never fully be understood

Reply:
![image](https://github.com/user-attachments/assets/7bf7f287-7354-4281-a0a5-0ba86c842b5a)
As far as we are concerned, the expectation is as follows.
1   2
4   5

The result of Prosemirror is like this.
1

2

4

5

I will change the rules to be the same as ProseMirror, and then reconsider whether to add a custom renderText method
Thank you for your suggestion.